src/config/translation.js

```diff
@@ -50,6 +50,9 @@ const tsObj = {
     'Runtime Selector missing value': '环境选择器缺失值',
     'Invalid s3 url, should be like s3://s3.pek3a.qingstor.com/op-repo':
       '无效的s3网址，应该是这样: s3://s3.pek3a.qingstor.com/op-repo',
+    'Verify that the information requested is incomplete!': '验证要求输入的信息不完整！',
```

Comment:
`Information to be verified is incomplete`

Reply:
ok.